Mage.Sets/src/mage/cards/r/ReturnTriumphant.java

```diff
@@ -0,0 +1,85 @@
+package mage.cards.r;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.CreateRoleAttachedTargetEffect;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.FilterCard;
+import mage.filter.common.FilterCreatureCard;
+import mage.filter.predicate.mageobject.ManaValuePredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.common.TargetCardInYourGraveyard;
+import mage.target.targetpointer.FixedTarget;
+
+import java.util.UUID;
+
+/**
+ * @author Susucr
+ */
+public final class ReturnTriumphant extends CardImpl {
+
+    private static final FilterCard filter = new FilterCreatureCard("card with mana value 3 or less");
```

Comment:
should have "creature" in text right? I know the card text is overwritten but this text will still be used in-game when selecting

Reply:
Oh missed that indeed.

Reply:
And "from your graveyard" too, as it can be missed and the TargetCardInYourGraveyard does not add that itself.